exercises/multiclass/ex_rnw/ic_sol_multiclass_01loss.Rnw

```diff
@@ -0,0 +1,70 @@
+\begin{enumerate}
+  \item 
+  	As seen in the \href{https://github.com/slds-lmu/lecture_sl/blob/main/slides-pdf/slides-advriskmin-classification-01.pdf}{0-1-Loss presentation, slide 2}, the discrete classifier that minimizes the risk $\hbayes$ (The Bayes optimal classifier) is:
+  	
+  	\begin{align*}
+  	  \hxbayes &=  \argmax_{l \in \Yspace } \ \ \underbrace{\P(y = l~|~ _{\xv = \xv})}_{\sim \mathrm{Unif} \{1,\ldots,x\} } \\
+  	  &=  \argmax \frac{1}{x} \cdot \I_{[1\leq l\leq x]}
```

Comment:
In the lecture, we use a different symbol for the indicator function, please use the same one here (and later in this solution)